### PR TITLE
add card, responsiveness, prism-js

### PR DESCRIPTION
**Pull request by iRohitGaur**

### CODE REVIEW

File: pages/components/button/button.css [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
+  font-size: 1.6rem;
   font-weight: 500;
   transition: all ease 0.5s;
 }
 .sui_btn:hover {
   cursor: pointer;
-  background: var(--PRIMARY-M2);
-  border: 2px solid var(--PRIMARY-M2);
+  /* background: var(--PRIMARY-M2);
+  border: 2px solid var(--PRIMARY-M2); */
```
Comment (zoef-yt) [inline pages/components/button/button.css]: Remove commented code

**Review by zoef-yt — APPROVED** [review]
Looks great. Good merge.

File: pages/components/button/button.css (hunk 1 above)
Comment (iRohitGaur) [reply pages/components/button/button.css]: Thanks for pointing it out. It had to be uncommented with a different value. Will fix.

Comment (iRohitGaur) [reply pages/components/button/button.css]: Updated!

**Review by srishtigautam21 — APPROVED** [review]
Your work is always up to mark.....good to merge

**Review by bhushannaikaroskar — APPROVED** [review]
Good to go

**Pull request merged**